src/templates/index.html

```diff
@@ -63,14 +63,16 @@
 
                     <div class="row">
                         <div class="input-field col s6">
-                            <input placeholder="For eg: 1" id="first_name" type="number" name="range_low" class="validate">
+                            <input placeholder="For eg: 1" id="first_name" type="number" name="range_low"
+                                class="validate">
                             <label for="first_name" style="color: black;font-size: 20px;">Low</label>
                             <span class="helper-text" data-error="wrong" data-success="right">Enter the lower range
                                 (inclusive)</span>
 
                         </div>
                         <div class="input-field col s6">
-                            <input placeholder="For eg: 8888" id="last_name" type="number" name="range_high" class="validate">
+                            <input placeholder="For eg: 8888" id="last_name" type="number" name="range_high"
+                                class="validate">
```

Comment:
Same here.
> Do we need to a new line here?
> _Trailing content of previous line._

Reply:
I'll make sure that these things don't happen from next time.
I'll correct them.